### PR TITLE
Add support for progress view insets and corner radius

### PR DESCRIPTION
**Pull request by k06a**
```objective-c
style.progressBarHorizontalInsets = 8.0;
style.progressBarCornerRadius = 1.0;
```
=>
img_9711

### CODE REVIEW

**Comment by k06a** [comment]
@jaydee3 why not to merge this PR?

File: JDStatusBarNotification/JDStatusBarNotification.m [hunk 1]
```diff
@@ -369,7 +369,8 @@ - (void)setProgress:(CGFloat)progress;
     CGFloat height = MIN(frame.size.height,MAX(0.5, self.activeStyle.progressBarHeight));
     if (height == 20.0 && frame.size.height > height) height = frame.size.height;
     frame.size.height = height;
-    frame.size.width = round(frame.size.width * progress);
+    frame.size.width = round((frame.size.width - 2*self.activeStyle.progressBarHorizontalInsets)*progress);
```
Comment (jaydee3) [inline JDStatusBarNotification/JDStatusBarNotification.m]: Add spaces here: `2 * self`

File: JDStatusBarNotification/JDStatusBarNotification.m [hunk 2]
```diff
@@ -369,7 +369,8 @@ - (void)setProgress:(CGFloat)progress;
...
+    frame.origin.x += self.activeStyle.progressBarHorizontalInsets;
```
Comment (jaydee3) [inline JDStatusBarNotification/JDStatusBarNotification.m]: Make this `frame.origin.x = self.activeStyle.progressBarHorizontalInsets;`

**Comment by jaydee3** [comment]
Will merge, if you apply the two tiny points.

**Comment by podkovyrin** [comment]
@jaydee3 done :)

**Comment by podkovyrin** [comment]
Will you release a new pod version?

**Comment by jaydee3** [comment]
done!

**Comment by k06a** [comment]
Awesome!

**Pull request merged**